src/js/components/jtable/JTable.jsx

```diff
@@ -0,0 +1,137 @@
+// @flow
+
+import React, { Component, PropTypes, Children } from 'react';
+
+export const TABLE_LEFT_RIGHT_PADDING = 24;
+export const TABLE_COLUMN_SPACING = 32;
+// !!!IMPORTANT!!! Don't change the above consts without changing jtable.less to match!
+
+export type ColumnDescription = {
+    name: string,
+    width: number,
+    isFlexible: boolean
+};
+
+type Props = {
+    className?: string,
+    children: ReactChildren,
+    columns: Array<ColumnDescription>,
+    style?: Object
+};
+
+/**
+ * Table component, rendered using divs and display:flex rather than HTML tables, because that gives us better control
+ * over layout, and allows us to do things like making a whole row an anchor, etc.
+ */
+export class JTable extends Component {
+
+    /**
+     * Just a simple helper
+     */
+    static column(width: number, name: string = '', isFlexible: boolean = false): ColumnDescription {
+        return {name, width, isFlexible};
+    }
```

Comment:
I had a little trouble following what `name` is actually used for in the code, unless it's just there for debugging the list of columns? If it's truly optional, perhaps putting `isFlexible` as the second param would be a small improvement, since I can see the width and flexibility being something folks will use frequently.